keyboards/merge/um80/keymaps/default/keymap.c

```diff
@@ -0,0 +1,184 @@
+/* Copyright 2021 duoshock
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include <stdio.h>
+
+enum layer_names {
+    _BASE,
+    _ONE,
+    _TWO,
+    _THREE
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/*
+ *          .---------------------------.          .-------------------------------.        .---.
+ *          |Esc|  1|  2|  3|  4|  5|  6|          |  7|  8|  9|  0|  -|  =|  Bsp  |        |EC |
+ *          |-------------------------.          .---------------------------------|        |---|
+ *          |Tab  |  Q|  W|  E|  R|  T|          |  Y|  U|  I|  O|  P|  [|  ]|  \  |        |Hom|
+ *   .---.  |-------------------------.          '---------------------------------|        |---|
+ *   |M0 |  |CapsL |  A|  S|  D|  F|  G|          |  H|  J|  K|  L|  ;|  :|   Retn |        |End|
+ *   |---|  |----------------------------.        '--------------------------------|  .---. '---'
+ *   |M1 |  |Shft    |  Z|  X|  C|  V|  B|          |  N|  M|  ,|  ,|  /|  Shift|     |Up |
+ *   |---|  |----------------------------|          |---------------------------' .-----------.
+ *   |M2 |  |Ctl |Gui |Alt |Fn0 |Space   |          |    Space|   Alt|   Ctl|     |Lef|Dow|Rig|
+ *   '---'  '----------------------------'          '-----------------------'     '-----------'
+ */
+
+[_BASE] = LAYOUT_rspace_2u_bksp(
```

Comment:
This wont compile as the LAYOUT macro doesnt exit in `um80.h`

Reply:
I've just updated it. Thanks!